### ISSUE TITLE
need ability to specify config file

### ISSUE THREAD

**Issue by 75lb** [lwsjs/local-web-server#44]
currently, `ws` merges config loaded from `package.json` and/or `.local-web-server.json` (depending on which exists). 

Implement a `--config-file` (or similar) option to enable the user to specify a config file name. This would be useful for A/B testing, comparing server results using config A against config B.

**Comment by 75lb** [comment]
this is resolved by the `--config-file` option in v2 which is in progress and [available for preview](https://github.com/lwsjs/local-web-server/tree/next).. any feedback welcome. 

Config files are now plain javascipt, giving you freedom to share and merge options however you like. The config file (`lws.config.js` by default) should look something like this:

```js
module.exports = {
  rewrite: [
    {
      from: '/resources/*',
      to: 'http://remote-api.org:8080/resources/$1'
    }
  ],
  directory: 'src',
  'log.format': 'none'
}
```

**Comment by 75lb** [comment]
fixed and released in v2.0.0
